src/save.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use flate2::write::GzEncoder;
+use flate2::Compression;
+use policy_evaluator::policy_fetcher::store::{PolicyPath, Store};
+use std::fs::File;
+
+// saves all policies in a tarball with the name provided as output.
+// policies must be inside the default store.
+pub(crate) fn save(images: Vec<&String>, output: &str) -> Result<()> {
+    let tar_gz = File::create(output)?;
+    let enc = GzEncoder::new(tar_gz, Compression::default());
+    let mut tar = tar::Builder::new(enc);
+
+    for image in images {
+        let store = Store::default();
+        let wasm_path = crate::utils::wasm_path(image.as_str())?;
+        let mut file = File::open(wasm_path)?;
```

Comment:
same here, add more context. This is even more important here because multiple policies are being read, the user must know which one failed

Reply:
https://github.com/kubewarden/kwctl/pull/348/commits/0798f9170407058434b66f02e205dfdf1f92ef7d